### DISCUSSION TITLE
Add configuration for tick labels

### DISCUSSION THREAD

**Discussion by jcobb88** [matoos32/jfreechart-builder#53] [Ideas]
The 'ChartBuilder' should offer a method to customize the tick labels, e.g. ChartBuilder#setTickLabelFormatting(SimpleDateFormat format). The format could be stored as a field and called upon the #build() method (e.g. ChartBuilder, line 279).
In this context a #setVerticalTickLabels would also be useful to allow more tick labels on the chart (e.g. call on NumberAxis in line 229 of the ChartBuilder).

**Comment by matoos32** [comment]
Thanks for the input. I'll have a look when I'm able to.

**Comment by matoos32** [comment]
Tracked in enhancement #61

**Comment by matoos32** [comment]
@jcobb88 I implemented enhancement #61 and you can access it as of [pull request #62](https://github.com/matoos32/jfreechart-builder/pull/62)

I also updated the README. I hope you find it useful. Let me know what you think.

**Reply by matoos32** [reply]
Please note a subsequent tech-debt [pull request #63](https://github.com/matoos32/jfreechart-builder/pull/63) that adds breaking changes due to renamings.
